### DISCUSSION TITLE
Git for Windows v2.47.0.windows.1

### DISCUSSION THREAD

**Discussion by gitforwindowshelper** [git-for-windows/git#5197] [Announcements]
Changes since Git for Windows v2.46.2 (September 24th 2024)

Git for Windows for Windows v2.47 drops support for Windows 7 and for Windows 8, as announced previously.

Please also note that the 32-bit variant of Git for Windows is deprecated; Its last official release [is planned for 2025](https://gitforwindows.org/32-bit.html).

### New Features

* Comes with [Git v2.47.0](https://github.com/git/git/blob/v2.47.0/Documentation/RelNotes/2.47.0.txt).
* Comes with the MSYS2 runtime (Git for Windows flavor) based on [Cygwin v3.5.4](https://inbox.sourceware.org/cygwin-announce/20240825195526.2571058-1-corinna-cygwin@cygwin.de/), which drops Windows 7 and Windows 8 support.
* The new, experimental `git backfill` command [was added](https://github.com/git-for-windows/git/pull/5172): It helps fetching relevant Git objects smartly in a partial, sparse clone.
* The new, experimental [`git survey` command was added](https://github.com/git-for-windows/git/pull/5174). This command is designed to help identify less-than-ideal data shape in monorepos, and it will likely see highly active development. Stay tuned!
* Comes with [Git Credential Manager v2.6.0](https://github.com/git-ecosystem/git-credential-manager/releases/tag/v2.6.0).

Filename | SHA-256
-------- | -------
Git-2.47.0-64-bit.exe | 30e8585fd7055eb1417ca4472dbcfcf65dd80c355613d3a6e115d695e7dc31bd
Git-2.47.0-32-bit.exe | 082fccce46d7d5b34a0fd146dbc21ef2c7597f468a0e9a41c3814536a04d6668
PortableGit-2.47.0-64-bit.7z.exe | 0b7fcd76902ebde5b4c00ebae597d7f65dff8c3dd0ae59f5059e1aaa3adace87
PortableGit-2.47.0-32-bit.7z.exe | b2def285b907ce1d47abd2df8df83df629b768defe08c1fcd4ad91582fc6606b
MinGit-2.47.0-64-bit.zip | 6b175d4675bfa7014ba649e3c0976da2a281d89bd9a096f1f41437f9995497a8
MinGit-2.47.0-32-bit.zip | 8d009e2e2d0d17f9c31255065e83183f3f03f15ed238aff9592da0213bedc176
MinGit-2.47.0-busybox-64-bit.zip | 805e3553de73225ea757d80676fafdde004a73b3ae2dac0954d64e87337ebc4f
MinGit-2.47.0-busybox-32-bit.zip | 705b0b547d327f012b91a85122cbc58189297ba2e34885609186613ba4874882
Git-2.47.0-64-bit.tar.bz2 | 63d07aa9593be64e0e2510f69b14beac85e5fb6d3dc66fc0a7e17b8dea0ccfe0
Git-2.47.0-32-bit.tar.bz2 | c13099b232d9e3ff59538bea4e55cc6738c554f5030ce93df8ede7e6044e23b3

<hr /><em>This discussion was created from the release <a href='https://github.com/git-for-windows/git/releases/tag/v2.47.0.windows.1'>Git for Windows v2.47.0.windows.1</a>.</em>

**Comment by dscho** [comment]
> [!IMPORTANT]
> Please note that this version has two known regressions:
> - Fetching/pushing via SSH [experiences frequent dead locks](https://github.com/git-for-windows/git/issues/5199).
> - Repacking in repositories with many tags [became a lot slower](https://github.com/git-for-windows/git/pull/5205).
>
> Both regressions have been fixed in the most recent [snapshot](https://wingit.blob.core.windows.net/files/index.html).

In anticipation for more bug fixes in the upstream Git project, and therefore a potential Git v2.47.1, I want to hold off from releasing a new Git for Windows version. Besides, I try not to release two Git for Windows versions within the same week, to avoid upgrade fatigue.

**Reply by dscho** [reply]
I released [Git for Windows v2.47.0(2)](https://github.com/git-for-windows/git/releases/tag/v2.47.0.windows.2) yesterday.
